tests/test_todos.py

```diff
@@ -0,0 +1,81 @@
+import pytest
```

Comment:
You need to add `pytest` and `selenium` to your list of dependencies (in requirements.txt)